pkg/prometheus/promcfg_test.go

```diff
@@ -12478,3 +12478,156 @@ func TestServiceMonitorSelectors(t *testing.T) {
 		})
 	}
 }
+
+func TestPodMonitorSelectors(t *testing.T) {
+	for _, tc := range []struct {
+		name                 string
+		golden               string
+		podMonitor           *monitoringv1.PodMonitor
+		serviceDiscoveryRole monitoringv1.ServiceDiscoveryRole
+	}{
+		{
+			name:                 "PodMonitor with Match Label Selector",
+			golden:               "PodMonitorObjectWithMatchLabelSelector.golden",
+			serviceDiscoveryRole: monitoringv1.EndpointsRole,
```

Comment:
serviceDiscoveryRole shouldn't make a difference in the generated config, right?

Reply:
Right, it's a copy and paste mistake 😅